mteb/cli.py

```diff
@@ -251,6 +253,56 @@ def add_run_parser(subparsers) -> None:
     parser.set_defaults(func=run)
 
 
+def potentially_add_cqadupstack_to_results(results: list[mteb.MTEBResults]) -> None:
+    """If all CQADupstack tasks are present in the results, combine them into a single CQADupstackRetrieval task and add it to the results."""
+    TASK_LIST_CQA = set(
```

Comment:
Instead of set([a, b, c]) why not {a, b, c}?

Reply:
yea probably better